osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -100,23 +94,21 @@ public OsuSliderBar()
                         CornerRadius = 5f,
                         Children = new Drawable[]
                         {
-                            LeftBox = new Box
+                            RightBox = new Box
                             {
                                 Height = 5,
                                 EdgeSmoothness = new Vector2(0, 0.5f),
-                                RelativeSizeAxes = Axes.None,
-                                Anchor = Anchor.CentreLeft,
-                                Origin = Anchor.CentreLeft,
+                                Anchor = Anchor.CentreRight,
+                                Origin = Anchor.CentreRight
                             },
-                            RightBox = new Box
+                            LeftBox = new Box
```

Comment:
Can you explain why these two were flipped in depth?

Reply:
Likely purely an accident that I didn't see.